BUILD

```diff
@@ -0,0 +1,8 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix", "gazelle")
+
+go_prefix("github.com/bazelbuild/bazel-integration-testing")
```

Comment:
`go_prefix` is deprecated. You shouldn't need it if you have `importpath` for all Go rules in the subtree below.

Reply:
Done

Reply:
Still here?